src/libxively/sft/xi_sft_logic.c

```diff
@@ -7,6 +7,10 @@
 #include <xi_sft_logic.h>
 #include <stddef.h>
 #include <xi_macros.h>
+#include <xi_debug.h>
+
+// #define XI_SFT_DOWNLOAD_BYTES_PER_FILE_CHUNK 1 * 1024
```

Comment:
yeah, this is the same I guess as it is with the doubling sized cbor encoder buffer, so here we should configure it from make command, don't we?